packages/tools/client-debugger/client-debugger-view/src/components/ValueView.tsx

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { ValueNodeBase } from "@fluid-tools/client-debugger";
+import { Accordion } from "./utility-components";
+
+/**
+ * {@link ValueView} input props.
+ */
+export interface ValueViewProps {
+	node: ValueNodeBase;
+}
+
+/**
+ * Render data with type {@link VisualNodeKind.ValueNode}.
+ */
+export function ValueView(props: ValueViewProps): React.ReactElement {
+	const { node } = props;
+
+	return (
+		<Accordion
```

Comment:
I don't think we need to use an Accordion for this, since there aren't any children. We should be able to render this like we do our unknown data views.